src/app/wallet/views/gas.nim

```diff
@@ -15,7 +16,11 @@ type
 const getGasPredictionsTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
   let
     arg = decode[GasPredictionsTaskArg](argEncoded)
-    output = %getGasPricePredictions()
+    response = status_wallet.getGasPrice().parseJson
+  var output = "0"
+  echo response
```

Comment:
I'd guess the echo should be removed and was used for debugging?

Reply:
oops :)